main.go

```diff
@@ -145,10 +145,16 @@ func main() {
 	}
 	opts := zap.Options{}
 	opts.Level = level
-	logger := zap.New(zap.UseFlagOptions(&opts))
+	logger := zap.New(zap.UseFlagOptions(&opts)).WithName("setup")
 	ctrl.SetLogger(logger)
 
-	setupLog = ctrl.Log
+	operatorNamespace, err := getOperatorNamespace()
+	if err != nil {
+		setupLog.Error(err, "cannot get the operator namespace")
+		os.Exit(1)
+	}
+	setupLog = ctrl.Log.WithValues("namespace", operatorNamespace)
```

Comment:
I don't think we should record the namespace in each log message. This namespace suppose to be same through the entire run of the operator and should be known to the admin before viewing the log file (`kubectl logs k4e-podname -n k4e-namespace`)

Reply:
You're right! Is there any other contextual information you think is worth adding?

Reply:
not for the setupLog